### ISSUE TITLE
Feel there needs to be some examples

### ISSUE THREAD

**Issue by Sternheart** [Zerbu/Mod-Constructor-5#81]
I have been trying to make a broadcaster for a trait, but there is no information on how to to do that one task especially with a timer which would be handy.

thank you.

**Comment by Finchy66** [comment]
Yes, I agree! It is so confusing and there are hardly any tutorials on youtube explaining more in-depth. :/ I'm having a similar problem but with proximity buffs.

**Comment by Finchy66** [comment]
Hi yes sort of, i know how proximity buffs work since I've made them plenty of times before but for some reason when I make a proximity buff and instead of "always run" I use the "interaction running condition" and so when my sim listens to any kind of music the proximity buff "Comforting music" is supposed to appear but for some odd reason it doesn't. The participant is targetsim but I've also tried changing it to actor and still nothing. :( I'm trying to make a trait but its just not going how i planned. 
ALSO IM SO SORRY I shouldn't be speaking of my issue i should be trying to resolve yours. I really do apologise.
